facp.md

```diff
@@ -0,0 +1,82 @@
+# FACP
+
+The FADT, of which the signiture is FACP, refers to the Fixed ACPI Description Table, and is pointed by the RSDT.
+The RSDT has entries that point to ACPI, FADT and more.
+The entries are 4 in number, and each of them are pointing to the tables.
+FACP is one of them, and could be found by strncmp.
+Here is the example code:
+```c
+if(strncmp(header->Signature, "FACP", 4)==0){
+	printf("FACP found\n");
+}
+```
+Once can find the address of it via comparing strings.
```

Comment:
Maybe something is missing probably you wante to say that once you found the signature you can get the address from that entry?

Reply:
typo

Reply:
lol